library/include/rocwmma/internal/pack_util_impl.hpp

```diff
@@ -104,6 +104,21 @@ namespace rocwmma
         using PackedT   = int32_t;
     };
 
+#if defined(__gfx940__)
```

Comment:
These are also just static properties. The datatypes should be available for all archs.
Reason is: we can support data load / store, just not mfma.